stable/nginx/nginx/templates/NetworkPolicy.yaml

```
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: nginx-{{ .Values.Instance }}-network-policy
+  labels:
+    app: {{ template "nginx.name" . }}
+    chart: {{ template "nginx.chart" . }}
+    release: {{ .Release.Name }}
+    instance: {{ .Values.Instance | quote }}
+spec:
+  podSelector: {}
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: 155.101.8.34/32 
```

```suggestion
        cidr: {{ .Values.NetworkPolicy.AllowedCIDR }}
```

Let's template this.